aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -316,6 +319,13 @@ private static boolean isMasterUserSecretStabilized(DBCluster dbCluster) {
         return MASTER_USER_SECRET_ACTIVE.equalsIgnoreCase(dbCluster.masterUserSecret().secretStatus());
     }
 
+    private static boolean isGlobalWriteForwardingStabilized(DBCluster dbCluster) {
```

Comment:
Can you add a test for the stabilization logic?

Reply:
Added BaseHandler tests for DBCluster